lms/djangoapps/grades/api/views.py

```diff
@@ -252,7 +247,7 @@ class GradesBulkAPIView(ListAPIView):
         * percent: A float representing the overall grade for the course
 
 
-    **Example GET Response**
+    **Example POST Response**
         {
           "course-v1:T3+T3+2014_T1": {
             "lidijarad": {
```

Comment:
please remove your username and use a dummy name, remember this is open source, peeps can hack your account with  your uername :)